integration/spark/src/main/scala/org/apache/spark/sql/catalyst/CarbonParserUtil.scala

```diff
@@ -957,6 +958,16 @@ object CarbonParserUtil {
       }
     }
 
+    // Validate LINE_SEPARATOR length
+    if (options.exists(_._1.equalsIgnoreCase("LINE_SEPARATOR"))) {
+      val line_separator: String = CarbonUtil.unescapeChar(
+        options.get("line_separator").get.head._2)
+      if (line_separator.isEmpty || line_separator.length > 2) {
```

Comment:
do we need to have validations for supported line separators also ?
because for example, If I configured 'a' , it will not fail now. better to fail.

Reply:
> do we need to have validations for supported line separators also ?
> because for example, If I configured 'a' , it will not fail now. better to fail.

Univocity parser allow any one or two characters to be the line separator, we can just do the same. User may want use 'a' as line separator.

Reply:
ok